modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/UserCard.java

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.sample.web.internal.display.context;
+
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.LabelItem;
+
+import java.util.List;
+
+/**
+ * @author Carlos Lancha
+ */
+public class UserCard
```

Comment:
Already discussed in the original PR https://github.com/liferay-frontend/liferay-portal/pull/434/commits/b95cc1c1b4cf6d0c4a1a09de8e017d50974823fb#r511909334

Reply:
> Already discussed in the original PR [b95cc1c#r511909334](https://github.com/liferay-frontend/liferay-portal/commit/b95cc1c1b4cf6d0c4a1a09de8e017d50974823fb#r511909334)

That link doesn't seem to lead to anything related. I tried searching in old PR, but everything is collapsed into resolved :slightly_smiling_face:  I'll take your word for it that this is fine :+1:

Reply:
Github being github... 

This is Chema's comment
> So, in the taglib-clay-sample-web we now have 2 different CardDisplayContext classes, one free, one implementing UserCard?
I think the convention for the latter is to name them UserCard and not DisplayContext, and then assign it as userCard="<%= userCard %> rather than by exploding the attributes. Maybe that's what's confusing me... 🤔

Reply:
I think @jbalsas meant `UserCard` suffix, for example `ViewMembershipRequestsUserCard`. In our context, it would be something like `SampleUserCard`. I don't care particularly what it's named, just that `UserCard` implies base class. I was confused for a bit going through code, assuming on first glance that is the case.

Reply:
He didn't talk about any suffix, he's clearly saying `userCard`.

```
userCard="<%= userCard %>
```

Anyway, this is not critical.

Reply:
There's a lot of context in here that's lost, so I can't be certain for sure, but yeah, I'd expect our class to be `SomethingUserCard extends UserCard`.  

I was puzzled by us having  `userCard = SomethingThatsDisplayContext`. If we do provide a model, why not extend it?

Reply:
Renamed and replaced all `ClayCardDisplayContext` usages for `<clay:user-card>`